packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-node.vue

```diff
@@ -27,6 +29,23 @@ const emit = defineEmits(['open-drilldown', 'append-input-port', 'update-state']
 const modelInput = props.node.inputs.find((input) => input.type === 'modelId');
 const isModelInputConnected = computed(() => modelInput?.status === WorkflowPortStatus.CONNECTED);
 
+watch(
+	() => modelInput?.value,
+	async () => {
+		if (modelInput?.value) {
+			const modelId = modelInput.value[0];
+			const modelConfigurations = await getModelConfigurationsForModel(modelId);
+			if (isEmpty(modelConfigurations)) {
+				// Create a model configuration if it does not exist
+				const model = await getModel(modelId);
+				if (model) {
+					postAsConfiguredModel(model);
+				}
+			}
+		}
+	}
+);
```

Comment:
I don't think this watcher is working properly, cause it is watching a non-reactive variable?

Reply:
IDK how it was working in the first place to be honest

Reply:
should be okay now...